src/pygbag/app.py

```diff
@@ -380,35 +384,19 @@ async def main_run(app_folder, mainscript, cdn=DEFAULT_CDN):
 """
     )
 
-    config_parse = configparser.ConfigParser()
-    # config_parse["DEPENDENCIES"] = {"ignoreDirs":[], "ignoreFiles":[]}
     if os.path.exists("pygbag.ini"):
```

Comment:
i think that should better moved right after args processed here https://github.com/pygame-web/pygbag/blob/9da289f8e2840d5435dc3eeec6d67c3cdb66f134/src/pygbag/app.py#L406
 so this way with a flag eg "--ini" if there is no pygbag.ini we can create one with command line memorized in main section. That way next call could be just `pygbag main.py`  and ini will serve as some basic project file